libraries/triumph-cmds/arguments.md

```diff
@@ -24,4 +24,15 @@ class MyClass extends BaseCommand {
     }
 }
 ```
-It is as simple as that to have arguments for your command.
\ No newline at end of file
+It is as simple as that to have arguments for your command.
+
+## @ArgName Annotation
+The `@ArgName` annotation is used to define the name of the argument should you not want the name of the argument
```

Comment:
Mention that the argument name is automatically the parameter name if compiled with `-parameters` compiler arg.